test/CMakeLists.txt

```diff
@@ -23,114 +22,95 @@ add_test(h265_profile_tier_level_parser_unittest h265_profile_tier_level_parser_
 add_executable(h265_common_unittest h265_common_unittest.cc)
 add_test(h265_common_unittest h265_common_unittest)
 target_link_libraries(h265_common_unittest PUBLIC h265nal)
-target_link_libraries(h265_common_unittest PUBLIC ${GTEST_LIBRARY} gtest_main)
-target_link_libraries(h265_common_unittest PUBLIC ${GMOCK_LIBRARY} gmock_main)
+target_link_libraries(h265_common_unittest PUBLIC GTest::gtest GTest::gtest_main)
```

Comment:
You're removing all references to GMOCK. I assume this builds in Linux and MacOS, but will confirm later

Reply:
It seems that gmock symbols are now included in gtest builds. This builds and links fine for me, as well as tests pass. In facts, in new builds of gtest, attempting to link both fails due to duplicate symbols, and gtest doesn't even build shared versions of gmock.